bin/test-db

```diff
@@ -8,9 +8,23 @@ from argparse import RawTextHelpFormatter
 
 # singerio images have required keyfile
 image_name = "singerio/mongo"
-image_tag = "4.2-bionic"
-# image_tag = "4.4-bionic"
-# image_tag = "5.0"
+
+# image_tag = "4.2-bionic"  # version 5.0.15 TODO Remove?
+# image_tag = "4.4-bionic"  # version 4.4.6
+# image_tag = "5.0"         # version 5.0.15
+image_tag = "6.0"           # version 6.0.4
```

Comment:
Does this mean to run older versions, you have to comment out `6.0`, uncomment the one you want, and then run `test_db start`?

Reply:
Yes, that's what I was doing.